Core/src/org/sleuthkit/autopsy/corecomponents/DataResultViewerTable.java

```diff
@@ -539,8 +549,43 @@ protected void setColumnWidths() {
 
                     int columnWidth = Math.max(valuesWidth, headerWidth);
                     columnWidth += 2 * margin + padding; // add margin and regular padding
-                    columnWidth = Math.min(columnWidth, columnWidthLimit);
 
+                    columnWidth = Math.min(columnWidth, columnWidthLimit);
+                    columnWidths.add(columnWidth);
+                    
+                    totalColumnWidth += columnWidth;
+                    
+                    if( columnWidth == columnWidthLimit) {
+                        cntMaxSizeColumns++;
+                    }
+                }
+                
+                // Figure out how much extra, if any can be given to the columns
+                // so that the table is as wide as outlineViewWidth. If cntMaxSizeColumns
+                // is greater than 0 divide the extra space between the columns 
+                // that could use more space.  Otherwise divide evenly amoung 
+                // all columns.
+                int extraWidth = 0;
+                
+                if (totalColumnWidth < outlineViewWidth) {
+                    if  (cntMaxSizeColumns > 0) {
+                        extraWidth = (int) ((outlineViewWidth - totalColumnWidth)/cntMaxSizeColumns);
+                    } else {
+                        extraWidth = (int) ((outlineViewWidth - totalColumnWidth)/columnWidths.size());
+                    }
+                }
+                
+                for(int column = 0; column < columnWidths.size(); column++) {
+                    int columnWidth = columnWidths.get(column);
+                    
+                    if(cntMaxSizeColumns > 0) {
+                        if(columnWidth >= 300) {
```

Comment:
Would we want to use the same int columnWidthLimit = (column == 0) ? 350 : 300; here so that when the first column size fell between was greater or equal to 300 but less than 350 we wouldn't wind up with a horizontal scroll bar by allocating it space we didn't account for in when counting columns that needed extra size and dividing the available extra size?

Reply:
I thought about that.  I could do that if you think it will make a difference.

Reply:
I don't know that it matters in the larger scheme of things, since sometimes we will have scroll bars and sometimes we won't either way.  Just caused me a moment while reading it to figure out if it would have any undesirable effects, which I don't think it will, but thought I would mention it because of that.